charts/sigstore-prober/templates/_helpers.tpl

```diff
@@ -31,4 +31,10 @@ Create args for sigstore prober components
 {{- if .Values.spec.args.writeProber }}
 - "-write-prober={{ .Values.spec.args.writeProber }}"
 {{- end }}
+{{- if .Values.spec.args.rekorRequests }}
+- "-rekor-requests={{ .Values.spec.args.rekorRequests }}"
+{{- end }}
+{{- if .Values.spec.args.fulcioRequests }}
+- "-fulcio-requests={{ .Values.spec.args.fulcioRequests }}"
+{{- end }}
```

Comment:
New flags. Related PR: https://github.com/sigstore/scaffolding/pull/395

Reply:
please update README.md and schema with these new values.

Reply:
> New flags. Related PR: [sigstore/scaffolding#395](https://github.com/sigstore/scaffolding/pull/395)

The chart definition and doc in https://github.com/sigstore/helm-charts/tree/main/charts/sigstore-prober needs to be updated to include these flags. Getting the json schema right is a little involved so if you want to leave these for me to make in another change, that's ok.

Reply:
I added json schema for the new custom endpoints, but I haven't really tested it. I am spinning up a local env to test now. I think I got the escaping wrong, so I'll verify locally.